CHANGES.rst

```diff
@@ -3,6 +3,7 @@
 
 - Update tests to adapt to changes in Redshift and SQLAlchemy
   (`Issue #140 <https://github.com/sqlalchemy-redshift/sqlalchemy-redshift/pull/140>`_)
+- Add official support for Python 3.7
```

Comment:
Could you add a link to this PR following the style of other entries?

Reply:
Done!